events/appsync.go

```diff
@@ -9,6 +9,20 @@ type AppSyncResolverTemplate struct {
 	Payload   json.RawMessage  `json:"payload"`
 }
 
+// AppSyncIdentity contains information about the caller. The shape of this section depends on the authorization type of your AWS AppSync API
```

Comment:
Seems that there'd need to be 2 types added, rather than the union of both shapes. https://docs.aws.amazon.com/appsync/latest/devguide/resolver-context-reference.html
* `AppSyncIAMIdentity`
* `AppSyncCognitoIdentity`

Reply:
Unmarshalling / marshalling `$context.identity` into the single type works either way, but I'll separate out the type if that's what's desired.